### DISCUSSION TITLE
Using Python in GitHub Pages

### DISCUSSION THREAD

**Discussion by deirdref88** [community/community#22187] [Other Feature Feedback, Questions, & Ideas]
<p>Hi! I’m new to GitHub Pages, so sorry if this is a silly question. I’m trying to transfer over a project that I already made (which uses HTML, CSS, Python, and Flask). But when I go to my website, it only seems to recognize the HTML parts of my project. How can I link the other languages into GitHub Pages? Thanks!</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! GitHub Pages can only deploy statically generated sites. It does support JavaScript to a certain degree but will not support Python and such.</p>
<p>See here for details: <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#static-site-generators" rel="noopener nofollow ugc">https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#static-site-generators</a></p>
